core/src/main/java/com/scalar/db/transaction/rpc/GrpcTransactionManager.java

```diff
@@ -98,7 +98,7 @@ public DistributedTransaction begin(String txId) throws TransactionException {
   private DistributedTransaction beginInternal(@Nullable String txId) throws TransactionException {
     return executeWithRetries(
         () -> {
-          GrpcTransactionOnBidirectionalStream stream = getBidirectionalStream();
+          GrpcTransactionOnBidirectionalStream stream = getStream();
```

Comment:
This kind of changes doesn't seem related to this PR, but it's actually a part of this PR?

Reply:
It's not directly related to this PR actually. It's just renaming the private method for unit tests.